p2p/conn/connection.go

```diff
@@ -425,22 +435,22 @@ func (c *MConnection) sendMsgPacket() bool {
 	if leastChannel == nil {
 		return true
 	} else {
-		// c.Logger.Info("Found a msgPacket to send")
+		// c.Logger.Info("Found a PacketMsg to send")
 	}
 
-	// Make & send a msgPacket from this channel
-	n, err := leastChannel.writeMsgPacketTo(c.bufWriter)
+	// Make & send a PacketMsg from this channel
+	_n, err := leastChannel.writePacketMsgTo(c.bufConnWriter)
```

Comment:
I started using _n to refer to "n" that is short lived in scope.  _n is *assigned* a number here without regard for any previous value (it isn't being accumulated) so I figured it would be consistent to just use _n for such uses.